src/providers/lambda/index.ts

```diff
@@ -0,0 +1,18 @@
+import { LAMBDA } from '../../globals';
+import {
+  chatCompleteParams,
+  completeParams,
+  responseTransformers,
+} from '../open-ai-base';
+import { ProviderConfigs } from '../types';
+import { LambdaAPIConfig } from './api';
+
+export const LambdaProviderConfig: ProviderConfigs = {
+  chatComplete: chatCompleteParams([], { model: 'Liquid-AI-40B' }),
+  complete: completeParams([], { model: 'Liquid-AI-40B' }),
```

Comment:
Is this the correct model ID? Their docs does not seem to mention this exact id anywhere. So I am not sure.

Reply:
I took it from their playground. They do support the `liquid` models as well.

Reply:
Got it